rpc.go

```diff
@@ -106,10 +125,14 @@ func (r *Rpc) ServeHTTP(w http.ResponseWriter, request *http.Request) {
 	if r.options.cors != "" {
 		w.Header().Set("Access-Control-Allow-Origin", r.options.cors)
 	}
+	if r.options.headers != "" {
+		w.Header().Set("Access-Control-Allow-Headers", r.options.headers)
+	}
+	if r.options.methods != "" {
+		w.Header().Set("Access-Control-Allow-Methods", r.options.methods)
+	}
 
 	if request.Method == http.MethodOptions && r.options.cors != "" { // Cors
-		w.Header().Set("Access-Control-Allow-Methods", "POST, OPTIONS")
-		w.Header().Set("Access-Control-Allow-Headers", "X-API-Key, Content-Type")
 		w.Header().Set("Access-Control-Max-Age", "86400")
```

Comment:
It would be great if you add settings for the max age like for methods and headers.